phaseshifts/gui/MainWindow.py

```
@@ -378,14 +356,10 @@ def updateModelUi(self, model=None):
         if isinstance(model, str):
             model = model.lower()
-            mtz = MTZ_model(
-                Unitcell(1, 2, [[1, 0, 0], [0, 1, 0], [0, 0, 1]]), atoms=[Atom("H")]
-            )
+            mtz = MTZ_model(Unitcell(1, 2, [[1, 0, 0], [0, 1, 0], [0, 0, 1]]), atoms=[Atom("H")])
             if model == "bulk":
                 tree = self.ui.treeWidgetBulk
-                mtz = eval(
-                    "self.%s" % model, {"self": self}, {}
-                )  # pylint: disable=eval-used
+                mtz = eval("self.%s" % model, {"self": self}, {})  # pylint: disable=eval-used
```

Ruff (0.14.10)

Use of possibly insecure function; consider using `ast.literal_eval`

(S307)